aviatrix/resource_aviatrix_gateway.go

```diff
@@ -432,6 +432,13 @@ func resourceAviatrixGateway() *schema.Resource {
 				Description:  "Price for spot instance. NOT supported for production deployment.",
 				RequiredWith: []string{"enable_spot_instance"},
 			},
+			"delete_spot": {
```

Comment:
"spot instance" support is only for internal test use. Should NOT be exposed in terraform state if not used.